extension.test.js

```diff
@@ -79,152 +127,267 @@ beforeEach(() => {
+  it('shows non breaking space in problems', () => {
+    mockDocument.text = 'non breaking space \u00a0'
+    activate(context)
+    expect(mockSetDiagnostics.mock.calls).toMatchSnapshot()
+  })
 
-it('shows left double quotation mark in problems', () => {
-  mockDocument.text = 'left double quotation mark \u201c'
-  activate(context)
-  expect(mockSetDiagnostics.mock.calls).toMatchSnapshot()
-})
+  it('shows left double quotation mark', () => {
+    mockDocument.text = 'left double quotation mark \u201c'
+    activate(context)
+    expect(mockSetDecorations.mock.calls).toMatchSnapshot()
+  })
 
-it('shows right double quotation mark', () => {
-  mockDocument.text = 'right double quotation mark \u201d'
-  activate(context)
-  expect(mockSetDecorations.mock.calls).toMatchSnapshot()
-})
+  it('shows left double quotation mark in problems', () => {
+    mockDocument.text = 'left double quotation mark \u201c'
+    activate(context)
+    expect(mockSetDiagnostics.mock.calls).toMatchSnapshot()
+  })
 
-it('shows right double quotation mark in problems', () => {
-  mockDocument.text = 'right double quotation mark \u201d'
-  activate(context)
-  expect(mockSetDiagnostics.mock.calls).toMatchSnapshot()
-})
+  it('shows right double quotation mark', () => {
+    mockDocument.text = 'right double quotation mark \u201d'
+    activate(context)
+    expect(mockSetDecorations.mock.calls).toMatchSnapshot()
+  })
 
-it('shows object replacement character', () => {
-  mockDocument.text = 'object replacement character \ufffc'
-  activate(context)
-  expect(mockSetDecorations.mock.calls).toMatchSnapshot()
-})
+  it('shows right double quotation mark in problems', () => {
+    mockDocument.text = 'right double quotation mark \u201d'
+    activate(context)
+    expect(mockSetDiagnostics.mock.calls).toMatchSnapshot()
+  })
 
-it('shows object replacement character in problems', () => {
-  mockDocument.text = 'object replacement character \ufffc'
-  activate(context)
-  expect(mockSetDiagnostics.mock.calls).toMatchSnapshot()
-})
+  it('shows object replacement character', () => {
+    mockDocument.text = 'object replacement character \ufffc'
+    activate(context)
+    expect(mockSetDecorations.mock.calls).toMatchSnapshot()
+  })
 
-it('shows multiple characters on multiple lines', () => {
-  mockDocument.text = `
-  zero width space \u200b\u200b\u200b
-  zero width non-joiner \u200c\u200c\u200c
-  non breaking space \u00a0\u00a0\u00a0
-  left double quotation mark \u201c\u201c\u201c
-  right double quotation mark \u201d\u201d\u201d
-  `
-  activate(context)
-  expect(mockSetDecorations.mock.calls).toMatchSnapshot()
-})
+  it('shows object replacement character in problems', () => {
+    mockDocument.text = 'object replacement character \ufffc'
+    activate(context)
+    expect(mockSetDiagnostics.mock.calls).toMatchSnapshot()
+  })
+
+  it('shows multiple characters on multiple lines', () => {
+    mockDocument.text = outdent`
+    zero width space \u200b\u200b\u200b
```

Comment:
Without outdent, these lines would all start with 4 spaces.  If this test got indented, the lines would start with 8.

Reply:
👍